configs/templates/ldap/ldap.yaml

```diff
@@ -3,14 +3,21 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: openldap
+spec:
```

Comment:
Do we need finalizers for this resource?

Reply:
When we are creating a Namespace, regardless of us mentioning the spec finalizer, it will automatically be kubernetes. So having it here or removing it seldom makes a difference.